datadog_checks_dev/datadog_checks/dev/tooling/commands/validate/metadata.py

```diff
@@ -481,21 +481,21 @@ def metadata(check, check_duplicates, show_warnings):
                     (echo_failure, f"{current_check}:{line} interval should be an int, found '{row['interval']}'.")
                 )
 
+            if 'curated_metric' in row and row['curated_metric']:
+                for curated_metric_type in row['curated_metric'].split('|'):
```

Comment:
Can we also validate that `row['curated_metric']` has unique elements, ie `cpu|cpu` is not allowed?